UefiTestingPkg/AuditTests/PagingAudit/UEFI/AArch64/PagingAuditAArch64.h

```diff
@@ -0,0 +1,12 @@
+#include <Library/ArmLib.h>
+#include <Chipset/AArch64Mmu.h>
+
+#define AARCH64_IS_TABLE(page, level)  ((level == 3) ? FALSE : (((page) & TT_TYPE_MASK) == TT_TYPE_TABLE_ENTRY))
+#define AARCH64_IS_BLOCK(page, level)  ((level == 3) ? (((page) & TT_TYPE_MASK) == TT_TYPE_BLOCK_ENTRY_LEVEL3) : ((page & TT_TYPE_MASK) == TT_TYPE_BLOCK_ENTRY))
+#define AARCH64_ROOT_TABLE_LEN(T0SZ)   (TT_ENTRY_COUNT >> ((T0SZ) - 16) % 9)
+#define AARCH64_IS_VALID  0x1
+#define AARCH64_IS_READ_WRITE(page)  (((page & TT_AP_RW_RW) != 0) || ((page & TT_AP_MASK) == 0))

```

Comment:
Do all of these macros need to be arch-specific?

For example, could a `IS_PAGE_RW()` macro take a page input and return a value without the arch in the name?